helm-charts/common/ui/templates/configmap.yaml

```diff
@@ -34,6 +34,9 @@ data:
   CHAT_URL: {{ .Values.BACKEND_SERVICE_ENDPOINT | quote }}
   {{- else if contains "faqgen-react-ui" .Values.image.repository }}
   VITE_FAQ_GEN_URL: {{ .Values.BACKEND_SERVICE_ENDPOINT | quote }}
+  {{- else if contains "videoqna-ui" .Values.image.repository }}
+  BACKEND_SERVICE_ENDPOINT: {{ tpl .Values.BACKEND_SERVICE_ENDPOINT . | quote }}
+  BACKEND_HEALTH_CHECK_ENDPOINT: {{ tpl .Values.BACKEND_HEALTH_CHECK_ENDPOINT . | quote }}
```

Comment:
I think there is no need to introduce a new variable BACKEND_HEALTH_CHECK_ENDPOINT, we can derive the health check endpoint by replace the last part of BACKEND_SERVICE_ENDPOINT with 'health_check'

Reply:
Done.